index.bs

```diff
@@ -85,7 +86,13 @@ Usage Example {#example}
 Terminology {#sec-terminology}
 ==============================
 
-<dfn export>Long task</dfn> refers to an event loop task that exceeds 50ms.
+<dfn export>Long task</dfn> refers to any of the following tasks whose duration exceeds 50ms:
```

Comment:
Maybe s/tasks/occurrences, or similar? Since they are not "tasks" in the HTML sense.

Reply:
Done